zipline/algorithm.py

```diff
@@ -986,6 +995,15 @@ def set_commission(self, commission):
             raise OverrideCommissionPostInit()
         self.commission = commission
 
+    @api_method
+    def set_symbol_lookup_date(self, dt):
+        """
+        Set the date for which symbols will be resolved to their sids
+        (symbols may map to different firms or underlying assets at
+        different times)
+        """
+        self.symbol_lookup_date = pd.Timestamp(dt)
```

Comment:
Looks like there isn't any.

```In [12]: wtz = Timestamp('2014-01-01', tz='US/Eastern')

In [13]: wtz.tz_convert("UTC") == Timestamp(wtz, tz='UTC')
Out[13]: True
```
I'm not sure why I wrote it that way originally.